### PR TITLE
[Term Entry] Add math-fabs method to math module on Codecademy docs

### PR DESCRIPTION
**Pull request by theosjones**
### Description

Added math.fab(s) method topic to the Math module for Python in Codecademy Docs.

### Issue Solved

Closes [Term Entry] Python math-module math.fabs() #5031

### Type of Change

Added new documentation to Codecademy Docs.

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-fabs/math-fabs.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+The **`math.fabs()`** function takes in a numeric data type and returns the absolute value as a float.
+
+## Syntax
+
+```pseudo
+math.fabs(n)
+```
+
+The `math.fabs()` function takes in a number value and returns its absolute value as a float. A non-negative number is returned and thus, a negative number is first negated. Unlike math.abs(), the value is always converted to a float value. Any other data type except a number will return a TypeError.
```
Comment (cigar-galaxy82) [inline content/python/concepts/math-module/terms/math-fabs/math-fabs.md:23]:
```suggestion
The `math.fabs()` function takes in a numeric value and returns its absolute value as a float. Unlike `math.abs()`, the value is always converted to a float value. Any other data type except a number will return a TypeError.
```

File: content/python/concepts/math-module/terms/math-fabs/math-fabs.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+The above code prints the following output:
+
+```shell
+154.2
+```
+
+## Codebyte Example
+
+The following examples run and use the `math.fabs()` function to return the absolute value of a number as a float:
```
Comment (cigar-galaxy82) [inline content/python/concepts/math-module/terms/math-fabs/math-fabs.md:43]:
```suggestion
Run the following code to understand how the `math.fabs()` function works:
```

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @theosjones thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @theosjones 😄 

The entry is ready to be merged! 🚀

**Pull request merged**